quick_task/routes/web.php

```diff
@@ -24,3 +25,5 @@
 Route::get('lang/{locale}', [LanguagesController::class, 'language']);
 
 Route::resource('staff', StaffsController::class);
+
+Route::resource('department', DepartmentController::class);
```

Comment:
```suggestion
Route::resource('departments', DepartmentController::class);
```
viết số nhiều theo chuẩn nha